config.sample.json

```diff
@@ -1,4 +1,5 @@
 {
     "base": "AUD",
-    "start_date": "2017-02-02"
+    "start_date": "2018-11-12",
```

Comment:
Would it make sense for this to be `<some recent date in YYYY-MM-DD format>`? That way we don't fall into the trap where the example config is constantly retreating into history? :)

Reply:
yes! time did NOT begin in November 2018.